examples/instance/maas/README.md

```diff
@@ -0,0 +1,68 @@
+# Infrakit MaaS Instance Plugin
+An example of [Infrakit](https://github.com/docker/infrakit) Instance Pluging for [Canonical MaaS](https://maas.io/) node.
+Purpose of this plugin is to support Bare metal deploy with Infrakit.
+## Go get 
+`$go get github.com/YujiOshima/infrakit-maas-plugin`
+
+## Get start
+Requires:
+* Needs virtualbox % sudo apt-get install virtualbox
+* Needs vagrant 1.6.x
+* Needs ansible installed on host machine
+* MaaS Server
+* API Key ( See https://maas.ubuntu.com/docs/maascli.html )
+* MaaS Nodes (You need all node be `Ready` state. Register and commision manually.)
+* Spec file ( example: maas-vanilla.yml )
+
+As this is only demo, we use virtual box. we should be use BareMetal Server Cluster with MaaS
+
+### Set up MaaS Cluster
+
+```
+$ git clone https://github.com/YujiOshima/vagrant-maas-in-a-box
+$ cd vagrant-maas-in-a-box
+$ vagrant up maas
+```
+
+In your browser visit http://localhost:8080/MAAS
+
+Default login and password: `admin` `pass`
+
+Import boot images in `Images` tab.
+
+! Wait for import images. It takes some time. !
+
+Set up nodes.
+
+```
+./setup-nodes.sh
```

Comment:
Sorry I was confused.  This is a file from your repo... I see now, so there's no need to include this.